Firmware/MVP/battsci.cpp

```diff
@@ -15,7 +15,19 @@
 uint8_t BATTSCI_state = STOPPED;
 
 uint8_t spoofedVoltageToSend = 0;
-int16_t spoofedCurrentToSend = 0; //JTS2doLater spoofed pack current can probably be int8_t (+127 A)
+int16_t spoofedCurrentToSend = 0;     //JTS2doLater spoofed pack current can probably be int8_t (+127 A)
+
+uint8_t LiBCM_SoC = 60;
+
+byte SoC_Bytes[] = {0x15, 0x50};      // SoC Bytes to send to MCM.  Index 0 is the upper byte and index 1 is the lower byte.
+byte temperature_Byte = 0x33;         // Temperature Byte to send to MCM.  0x33 is +21 Degrees C.
+uint16_t calculatedSoC = 500;
+uint16_t oldCalculatedSoC = 500;
+
+bool initializeSoC = true;
+
+uint8_t SoCUpdateDelayFrameIncrement = 17;	// How many frames between SoC updates to MCM?  OEM BCM usually does 17 but sometimes does 16
```

Comment:
Yes if we can get Gauge SoC reliably updating without this then it can be removed later.  I have this here just in case the MCM does expect SoC sent from the BCM to change only once every 16 or 17 frames.

Reply:
comment it out until we know it's actually required.  I don't believe it will be... I think it's actually the 'magic' frames that occur at each keyON event.